### PR TITLE
Update vulnerability.md

### PR DESCRIPTION
**Pull request by msbemba**
added another method for machine and saoftware

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5930

File: microsoft-365/security/defender-endpoint/vulnerability.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Method|Return Type|Description
 [Get all vulnerabilities](get-all-vulnerabilities.md)|Vulnerability collection|Retrieves a list of all the vulnerabilities affecting the organization
 [Get vulnerability by Id](get-vulnerability-by-id.md)|Vulnerability|Retrieves vulnerability information by its ID
 [List devices by vulnerability](get-machines-by-vulnerability.md)|MachineRef collection|Retrieve a list of devices that are associated with the vulnerability ID
+[List vulnerabilities by machine and software](get-all-vulnerabilities-by-machines.md)|Retrieves a list of all the vulnerabilities affecting the organization per machine and software.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/vulnerability.md]:
```suggestion
[List vulnerabilities by machine and software](get-all-vulnerabilities-by-machines.md)|Vulnerability collection|Retrieves a list of all the vulnerabilities affecting the organization per machine and software
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change if you agree with it. Many thanks.

File: microsoft-365/security/defender-endpoint/vulnerability.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/vulnerability.md]: I fixed it by making it as Vulnerability

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**